Web Forum/Models/User.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Web_Forum.Models
+{
+    public class User
+    {
+        [Key]
+        public int id { get; set; }
+        [Required]
+        public string email{ get; set; }
+        [Required]
+        public string password{ get; set; }
+        public string nickname{ get; set; }
```

Comment:
Wouldn't be better that nickname was required? Then we could show authors of topics and comments. Or show the information about which user is logged. I know we coud us other identifiers like emaile or id but I think that nickname would be the best for this purpous.

Reply:
It's required now.